src/main.go

```diff
@@ -131,10 +153,41 @@ func init() {
 	}
 }
 
+func init() {
+	// init dir GANTED_LOG_DIR
+	gantedLogDir := getEnv("GANTED_LOG_DIR", "/var/log/ganted")
+	if _, err := os.Stat(gantedLogDir); os.IsNotExist(err) {
+		os.Mkdir(gantedLogDir, 0755)
+	}
+}
+
+func setFileLoggerOutput(logger *log.Logger, filePath string) error {
+	if filePath == "" {
+		return nil
+	}
+	file, err := os.OpenFile(filePath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644)
+	if err != nil {
+		log.Fatalf("[ERR] Open log file: %s", err)
+		return err
+	}
+	logger.SetOutput(file)
```

Comment:
Consider fetching the previous writer and close it if valid. Example: [iBug/fakessh](https://github.com/iBug/fakessh/blob/8d175d6cc1e002a6e921059a98e676272254bc64/main.go#L48)